### PR TITLE
Add 'tiny_process_library' package

### PR DESCRIPTION
**Pull request by Bjoe**
* I've followed [this guide](https://cpp-pm-hunter.readthedocs.io/en/latest/creating-new/update.html)
  step by step carefully. **Yes**

* I've tested this package remotely and have excluded all broken builds.
  Here is the links to the Travis/AppVeyor with status "All passed":

  * https://ci.appveyor.com/project/Bjoe/hunter-t1t06/builds/29537745
  * https://travis-ci.org/Bjoe/hunter/builds/624972285

### CODE REVIEW

File: cmake/projects/tiny-process-library/hunter.cmake [hunk 1]
```diff
@@ -11,9 +11,9 @@ hunter_add_version(
     VERSION
     2.0.2
     URL
-    "https://gitlab.com/eidheim/tiny-process-library/-/archive/v2.0.2/tiny-process-library-v2.0.2.tar.gz"
+    "https://github.com/Bjoe/tiny-process-library/archive/624c2b61a32cb1544ec34f73bf1c5d87a8e2f19e.zip"
```
Comment (Bjoe) [inline cmake/projects/tiny-process-library/hunter.cmake]: Need merge of PR https://github.com/cpp-pm/tiny-process-library/pull/1

Comment (rbsheth) [reply cmake/projects/tiny-process-library/hunter.cmake]: @Bjoe Merged and released: https://github.com/cpp-pm/tiny-process-library/releases/tag/v2.0.2-p0

**Comment by rbsheth** [comment]
@Bjoe Should the docs be updated too now? Since `Threads` should be included through CMake? https://cpp-pm-hunter.readthedocs.io/en/latest/packages/pkg/tiny-process-library.html

File: cmake/projects/tiny-process-library/hunter.cmake (hunk 1 above)
Comment (Bjoe) [reply cmake/projects/tiny-process-library/hunter.cmake]: Done

**Comment by Bjoe** [comment]
> @Bjoe Should the docs be updated too now? Since `Threads` should be included through CMake? https://cpp-pm-hunter.readthedocs.io/en/latest/packages/pkg/tiny-process-library.html

Sorry I didn't get you.... you mean, add a comment in the doc that `Threads` is mandatory? Or you mean that `Threads` isn't needed because should be added as dependency in the cmake config modul .... I should test this.

**Comment by Bjoe** [comment]
Wait before merge. Test on travis and appveyor are running. I will inform you if all is fine.

**Comment by Bjoe** [comment]
Here is the links to the Travis/AppVeyor with status "All passed":
 * https://travis-ci.org/Bjoe/hunter/builds/626166029
 * https://ci.appveyor.com/project/Bjoe/hunter-tdmm2/builds/29596786

File: examples/tiny-process-library/CMakeLists.txt [hunk 2]
```diff
@@ -10,10 +10,9 @@ include("../common.cmake")
 project(tiny-process-library)
 
 # DOCUMENTATION_START {
-find_package(Threads REQUIRED)
 hunter_add_package(tiny-process-library)
-find_package(tiny-process-library REQUIRED)
+find_package(tiny-process-library CONFIG REQUIRED)
 
 add_executable(boo boo.cpp)
-target_link_libraries(boo PUBLIC tiny-process-library::tiny-process-library ${CMAKE_THREAD_LIBS_INIT})
```
Comment (rbsheth) [inline examples/tiny-process-library/CMakeLists.txt]: Removing `Threads` here is what I meant. Glad you figured it out 😅

**Comment by rbsheth** [comment]
Released: https://github.com/cpp-pm/hunter/releases/tag/v0.23.239

**Pull request merged**